ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/UserInterface/01c_AccountSheet.md

# Account sheet

> The *ACCOUNT SHEET* tab displays a chronologically ordered list of all movements in a specific account. The information displayed in the account sheet view is read-only.
> 
> 
> ## Account sheet view without account selection
> 
> *Accounting > Post > Tab ACCOUNT SHEET*
> 
> ![Account sheet - no account selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheetNoAccount.png "[Account sheet - no account selected]")
> 
> When clicking the *ACCOUNT SHEET* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the details of the account specified in the *Contra Account* field are displayed in the *ACCOUNT SHEET* tab. If the *Contra Account* field is empty, the details of the account specified in the *Account* field are displayed.
> 
> For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).
> 
> 
> ## Account sheet view with account selection
> 
> *Accounting > Post > Tab ACCOUNT SHEET*
> 
> ![Account sheet](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheet.png "[Account sheet]")

Reviewer: ```suggestion

When clicking the *ACCOUNT SHEET* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the details of the account specified in the *Contra Account* field are displayed in the *ACCOUNT SHEET* tab. If the *Contra Account* field is empty, the details of the account specified in the *Account* field are displayed.

For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).

## Account sheet view with account selection

*Accounting > Post > Tab ACCOUNT SHEET*

![Account sheet](../../Assets/Screenshots/RetailSuiteAccounting/Book/AccountSheet/AccountSheet.png "[Account sheet]")

The *ACCOUNT SHEET* tab displays a chronologically ordered list of all movements in a specific account. All fields are read-only.    
An account and/or contra account must be selected to display the corresponding details. Otherwise, the notice *No account selected* is displayed.    
You can either select an account in the *POSTING* tab and switch to the *ACCOUNT SHEET* tab or directly enter an account and/or contra account number in the input line in the bottom. For detailed information about accessing an account sheet, see [Access the account sheet](../Operation/02_ReviewAccount.md#access-the-account-sheet).   
If a contra account number is specified in the *Contra account* field of the input line, the account sheet for the corresponding contra account is displayed. If no contra account number is entered, but an account number is specified in the *Account* field of the input line, the account sheet for the corresponding account is displayed. 
```

> - *Fiscal year XXXX/Month X/Journal X*  
> This text displays the fiscal year, month and journal currently selected. This information is read-only.
> 
>   > [Info] If the fiscal year or the posting period need to be changed, click the [FY/MONTH/JOURNAL] button to display the select fiscal year window. For detailed information, see [Fiscal year](./00a_FiscalYear.md) and [Select a fiscal year](../Operation/01_SelectFiscalYear.md).
> 
> - ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
> Click this arrow to open the *Post extras* side bar. For detailed information about searching in the *Accounting* module, see [Search in accounting](../Operation/14_SearchAccounting.md).

Reviewer: ```suggestion
```

> - ![Previous](../../Assets/Icons/Previous.png "[Previous]")  *Scroll* ![Next](../../Assets/Icons/Next.png "[Next]")  
> Your can use the arrow buttons to scroll through the available accounts to search for a specific one. Alternatively, you can enter an account number in the input line.

Reviewer: ```suggestion
- ![Previous](../../Assets/Icons/Previous.png "[Previous]") (Previous)
  Click this button to display the account sheet of the previous available account number.

- ![Next](../../Assets/Icons/Next.png "[Next]") (Next) 
  Click this button to display the account sheet of the next available account number.
```

> - *Account sheet for account (account number) (account name)*  
> The number and name of the selected account are displayed.

Reviewer: ```suggestion
**Account sheet for account**   

The list displays all postings for the selected account whose number and name are specified in the list's title. All fields are read-only.
```

- *Date*  

> This column displays the posting date. The valid date format is DDMMYY.

Reviewer: ```suggestion
   Posting date in the format DDMMYY.
```

  > [Info] This is only applicable if at least one cost unit has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).

- *Debit*  
This column displays the debit amount of the selected account.

- *Credit*  
This column displays the credit amount of the selected account.

- *(Unnamed column)*
This column displays a ![B](../../Assets/Icons/Beleg.png "[B]") icon if a receipt has been linked to the posting. When clicking the icon, the linked receipt is displayed in the *Attached document* window. For detailed information, see [Attached document](#attached-document).

- *Text*  
This column displays the posting text, if any has been entered.

- *Total turnover (currency)*  
This field displays the total turnover of the selected account and its currency.

- *Balance (currency)*  
This field displays the balance of the selected account and its currency.

- *All months*  
Click this radio button to display the postings contained in the account sheet for all months.

- *Only this month*  
Click this radio button to display the postings contained in the account sheet only for the selected month.

- *Show (all/the last [XXX]) postings*    

> Click the drop-down list to select the preferred display filter.

Reviewer: vgl. 01a_Bookings

### Attached document

> 

Reviewer: vgl. 01_Book
-> vielleicht reicht auch ein Verweis auf das entsprechende Kapitel

*Accounting > Post > Tab ACCOUNT SHEET > Select icon B*

![Attached document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachedDocument.png "[Attached document]")

- ![Previous](../../Assets/Icons/Previous.png "[Previous]") *Document* ![Next](../../Assets/Icons/Next.png "[Next]")   
These buttons can be used to scroll through the documents. In this case, they do not have a real function, as there is only one document (or receipt) assigned to a posting.

- ![First page](../../Assets/Icons/FirstPage02.png "[Previous]") ![First page](../../Assets/Icons/Previous.png "[Previous]") *Page [x]* ![Next](../../Assets/Icons/Next.png "[Next]")  
The current page number is displayed. These buttons can be used to scroll through the pages. In this case, they do not have a real function, as there is only one document (or receipt) assigned to a posting.

- (-) *Zoom* (+)  
Click the -/+ buttons to zoom in and out of the document view.

- *Download file*  
Click this button to download the document.

- [CANCEL ASSIGNMENT]  
Click this button to undo the receipt assignment if necessary.

- [CLOSE]  

> Click this button to close the window.

Reviewer: Verweis auf input line
